MapboxNavigation.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -69,8 +69,8 @@
         "repositoryURL": "https://github.com/mapbox/MapboxGeocoder.swift.git",
         "state": {
           "branch": null,
-          "revision": "de735f4611f901acfe8295e9c42ca1eae54a7315",
-          "version": "0.13.0"
+          "revision": "91fb908bb57c0fcddc0062b5d64ea2002e0ea789",
+          "version": "0.14.0"
```

Comment:
Also update Package.resolved in the root directory by opening the Package.swift in Xcode.